### PR TITLE
WT-5547 Disable tests for columnar store that require rollback to stable support

### PR DESCRIPTION
**Pull request by kommiharibabu**
With current implementation of rollback to stable, the column store
types are not working. Comment them in the tests until the column
store is fixed.

### CODE REVIEW

**Comment by agorrod** [comment]
Should the title of this PR use disabling instead of enabling?

File: test/suite/test_txn07.py [hunk 1]
```diff
@@ -217,6 +210,14 @@ def test_ops(self):
         csmall = stat_cursor[stat.conn.log_compress_small][2]
         stat_cursor.close()
 
+        #
```
Comment (kommiharibabu) [inline test/suite/test_txn07.py]: Running the wt utility before collecting the statistics, the stats are reset. The connection is getting closed as part of running the wt utility is the cause of the problem. Moving it down after collecting the statistics doesn't have functionality problem and it fixes the test.

@sulabhM Do you have any idea?

File: test/suite/test_durable_rollback_to_stable.py [hunk 2]
```diff
@@ -44,7 +44,8 @@ class test_durable_rollback_to_stable(wttest.WiredTigerTestCase, suite_subproces
     keyfmt = [
         ('row-string', dict(keyfmt='S')),
         ('row-int', dict(keyfmt='i')),
-        ('column-store', dict(keyfmt='r')),
+    # The commented columnar tests needs to be enabled once rollback to stable for columnar is fixed.
```
Comment (sulabhM) [inline test/suite/test_durable_rollback_to_stable.py]: Can you create a ticket to enable these or if there is already one refer that ticket in the comment here.

File: test/suite/test_txn07.py (hunk 1 above)
Comment (sulabhM) [reply test/suite/test_txn07.py]: I do not know why this changed, there hasn't been any changes to printlog. Stats collection should work as before.

Comment (kommiharibabu) [reply test/suite/test_txn07.py]: I understood the reason behind passing the test before durable history. Earlier also the connection is getting closed and opened but as part of the connection open, WT will create the LAS file, due to which the log records are written and those are compressed and stats are updated. With DH, there are no las file create and it leads to no log records and it fails. Moving the `runWT` later after stats collection is proper way to fix the test and its expectation.

File: test/suite/test_durable_rollback_to_stable.py (hunk 2 above)
Comment (kommiharibabu) [reply test/suite/test_durable_rollback_to_stable.py]: Done. Added Jira ticket number WT-5548

**Pull request merged**